src/main/java/mousio/etcd4j/transport/EtcdNettyConfig.java

```diff
@@ -122,6 +130,40 @@ public boolean isManagedEventLoopGroup() {
   }
 
   /**
+   * Set a custom timer use to retry failed request
+   *
+   * @param timer custom timer
+   * @param managed whether timer will be closed when etcd client close, true represent yes
+   * @return itself for chaining.
+   */
+  public EtcdNettyConfig setTimer(HashedWheelTimer timer, boolean managed) {
+    if (this.managedTimer) { // if i manage it, close the old when new one come
+      this.timer.stop();
+    }
+    this.timer = timer;
+    this.managedTimer = managed;
+    return this;
+  }
+
```

Comment:
please add a setTimer(HashedWheelTimer timer) which ends up calling setTimer(timer, true) for consitenci with setEventLoopGroup